src/Faker/Provider/fa_AF/Text.php

```diff
@@ -0,0 +1,546 @@
+<?php
+
+namespace Faker\Provider\fa_AF;
+
+class Text extends \Faker\Provider\Text
+{
+    /**
+     * generates text string in arabic
+     *
+     * @example 'از تاریخ‌الشعرا را بکوبند روی نبش دیوار کوچه‌شان. تابلوی مدرسه.'
+     * @param  integer                   $maxNbChars
+     * @param  integer                   $indexSize
+     * @return string
+     * @throws \InvalidArgumentException
+     */
+    public function realText($maxNbChars = 200, $indexSize = 2)
```

Comment:
why do you need to override the original method?

Reply:
The added regular expression almost includes all the required and common characters in Persian/Dari text including different varieties of quotes and apostrophes, digits in Arabic and Persian formats,  zero width space characters, and others.